src/components/StatsNumbers/style.module.scss

```diff
@@ -0,0 +1,34 @@
+@use '../../styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  text-align: right;
+  gap: 2.5rem;
+  margin: 0 auto;
+  width: 80%;
+
+  .title {
+    @media screen and (max-width: vars.$breakpoint-md) {
+      font-size: 2.25rem !important;
+      line-height: 2.625rem !important;
+    }
+  }
+
+
+  .statNumber1 {
+    font-size: 1.75em;
```

Comment:
Change em to rem since you're not changing the font sizing for a parent container